D-6 reeditado con las nuevas rrss

BiggsBottor opened the pull request:
a ver si ahora va guay

BiggsBottor committed:
D-6 reeditado con las nuevas rrss

Mtllns reviewed (commented):
Eliminar scroll horizontal.
Eliminar simbolos de las listas.
Una etiqueta div de más.
Revisar class .ajusted

Mtllns commented on cera.html:
```diff
@@ -297,78 +341,120 @@ <h5 class="mr-10 text-justify mr-5 grey">PLAN YOUR VISIT
+                        <li class="nav-item"><a class="rrss text-light nav-link px-2" href="#"><i class="fab fa-twitter"></i></a></li>
+                        <li class="nav-item"><a class="rrss text-light nav-link px-2" href="#"><i class="fab fa-youtube"></i></a></li>
+                        <li class="nav-item"><a class="rrss text-light nav-link px-2" href="#"><i class="fab fa-linkedin-in"></i></a></li>
+                    </ol>
```
Cambiar etiqueta `<ol>` por `<ul>` y comprobar que no se muestren los numeros por defecto no los puntos.

Mtllns commented on cera.html:
```diff
@@ -297,78 +341,120 @@ <h5 class="mr-10 text-justify mr-5 grey">PLAN YOUR VISIT
+        <footer class="adjusted">
+            <div class="footer_1 d-flex justify-content-around flex-wrap pt-2">
+                <div class="row">
+                    <ol class="my_nav col-12 nav_footer navbar small align-self-center">
+                        <li class=" nav-item list-inline-item"><a class="text-light nav-link" href="#">Buy tickets</a></li>
+                        <li class=" nav-item list-inline-item text-warning">|</li>
+                        <li class=" nav-item list-inline-item"><a class="text-light nav-link" href="#">Plan your visit</a></li>
+                        <li class=" nav-item list-inline-item text-warning">|</li>
+                        <li class=" nav-item list-inline-item"><a class="text-light nav-link" href="#" onclick="openModal()">contact</a></li>
+                    </ol>
```
Cambiar etiqueta `<ol>` por `<ul>` y comprobar que no se muestren los numeros por defecto no los puntos.

Mtllns commented on cera.html:
```diff
@@ -121,6 +121,46 @@
+        .adjusted {
+            width: 100vw;
+            margin-right: -15px;
+            margin-left: -15px;
```
Ojo con esta clase. el width da problemas en el lado derecho.

Mtllns commented on cera.html:
```diff
@@ -297,78 +341,120 @@ <h5 class="mr-10 text-justify mr-5 grey">PLAN YOUR VISIT
                 </div>
             </div>
         </div>
-
```
Recuerda que por aquí falta el cierre de la etiqueta `<div class="container-fluid">` Lo hemos estado mirando Jonathan.

Mtllns commented on cera.html:
```diff
@@ -297,78 +341,120 @@ <h5 class="mr-10 text-justify mr-5 grey">PLAN YOUR VISIT
+                        <li class="nav-item"><a class="rrss text-light nav-link px-2" href="#"><i class="fab fa-twitter"></i></a></li>
+                        <li class="nav-item"><a class="rrss text-light nav-link px-2" href="#"><i class="fab fa-youtube"></i></a></li>
+                        <li class="nav-item"><a class="rrss text-light nav-link px-2" href="#"><i class="fab fa-linkedin-in"></i></a></li>
+                    </ol>
```
Más etiquetas `<ol>` probemos cambiar a `<ul>`

Mtllns commented on cera.html:
```diff
@@ -297,78 +341,120 @@ <h5 class="mr-10 text-justify mr-5 grey">PLAN YOUR VISIT
+        </footer>
+    </div>
+    <!-- FOOTER -->
+    </div>
```
Por aquí anda el conflicto de cierre de etiqueta `<div>` El que hemos corregido en tu PC, Jonathan.

BiggsBottor committed:
Add files via upload

Revisados:
- Eliminar scroll horizontal. (hecho, +info a continuación)
- Eliminar símbolos de las listas. (hecho)
- Una etiqueta div de más. (hecho. por lo visto he tenido que revisar los divs de las secciones anteriores porque afectaban al comportamiento del footer)
- Revisar class .ajusted (hecho, eliminada)
